### ISSUE TITLE
ropensci's `pkgcheck` action

### ISSUE THREAD

**Issue by pat-s** [cynkra/fledge#243]
https://ropensci.org/blog/2022/02/01/pkgcheck-action/

Might work well within `pre_release()` from the `f-27-automation` branch.

**Comment by krlmlr** [comment]
Are you suggesting to run pkgcheck locally as part of `pre_release()` ?

**Comment by pat-s** [comment]
As an action, as advertised.

**Comment by krlmlr** [comment]
What would the integration look like?

**Comment by pat-s** [comment]
Add WF YML during `pre_release()`, remove in `post_release()`. Haven't put much thought into it yet, though.

**Comment by krlmlr** [comment]
You can enable workflows on `cran-*` branches only, fledge won't need to do anything here. It seems worth documenting this pattern.

**Comment by pat-s** [comment]
Yes, I am aware. My idea was to reduce the amount of static YAML files and only have this action definition existing in the branch on purpose. 

I think there might be potential benefits to only have yaml files present that continuously do action on the current repo state. Otherwise one might loose overview which action actually runs when. Also given that `pre_release()` is doing similar "set up and remove" things anyhow.

Just a vague idea though, no strong opinion...

**Comment by maelle** [comment]
> Otherwise one might loose overview which action actually runs when. 

Off-topic but this sounds like a good idea for some sort of browser extension @pat-s (to get a diagram or so when clicking on the .github folder in the GitHub interface) :thinking:
